xbps-src

```diff
@@ -689,19 +689,20 @@ fi
 
 check_build_requirements
 
-if [ -z "$IN_CHROOT" ]; then
-    trap 'exit_func' INT TERM
-    if [ -n "$XBPS_DISTFILES_MIRROR" ]; then
-        setup_distfiles_mirror
-    fi
-fi
 #
 # Read funcs from helpers
 #
 for f in ${XBPS_SHUTILSDIR}/*.sh; do
     [ -r "$f" ] && . $f
 done
 
+if [ -z "$IN_CHROOT" ]; then
+    trap 'exit_func' INT TERM
+    if [ -n "$XBPS_DISTFILES_MIRROR" ]; then
+        setup_distfiles_mirror
+    fi
+fi
+
```

Comment:
At least from how the diff is displayed, I'd say the correct commit message would be `move msg_warn usage to after it's sourced` :p

Reply:
commit message is for why, not what :p